src/SvgVueR.vue

```diff
@@ -34,10 +34,14 @@
 
             <!-- resizing indicator -->
             <rect :x="width-10" :y="height-10" width="10" height="10"
-                @mousedown.stop="resizeStart" @mousemove.stop="resize"
-                @mouseup.stop="resizeEnd" @mouseleave="resizeEnd">
+                @mousedown.stop="resizeStart">
             </rect>
         </svg>
+        <div v-if="resizing" class="resizer" :style="{top:height + 'px', left:width + 'px'}"
```

Comment:
I'm now wondering: why move the "resizer" `div` ? After all, it already covers the whole view (as of your next commit in this branch) so all mouse movements will be inside this "resizer" `div` ( until you exit the window, in which case `@mouseleave` is triggered ). And so `resize` should be called in any case.

Reply:
Right.  Think I did that so it would overflow the main div container and contain that resizer trigger proportionally. But you're right. it should just be the whole screen. I'll update that.

Reply:
Should be all good now.